docs/requirements.txt

```diff
@@ -0,0 +1,6 @@
+atari_py==0.1.7
```

Comment:
Do we need to include these into docs? If not, we might be able to exclude it as well.

Reply:
I can remove atari_py. The other libs are definitely needed. They will not be included into the docs. But when readthedocs try to import our python files, they are needed.